source/include/ota.h

```diff
@@ -688,17 +688,17 @@ void otaAgentTask( const void * pUnused );
 /**
  * @brief Get the number of OTA message packets received by the OTA agent.
  *
- * @note Calling @ref OTA_AgentInit will reset this statistic.
+ * @note Calling @ref OTA_Init will reset this statistic.
  *
  * @return The number of OTA packets that have been received but not
  * necessarily queued for processing by the OTA agent.
  */
-uint32_t OTA_GetPacketsReceived( void );
+uint32_t OTA_GetStatistics( void );
```

Comment:
The OTA_GetStatistics is supposed to return all four params and not just packest received.

Reply:
Done,  change in https://github.com/divekarshubham/ota-for-aws-iot-embedded-sdk-1/commit/c3d146e6feacdaab0777e2a4a6391205088784d5